src/socket-proxy/socket-proxyd.c

```diff
@@ -597,6 +647,14 @@ static int parse_argv(int argc, char *argv[]) {
 
                         break;
 
+                case ARG_EXIT_IDLE:
+                        r = parse_sec(optarg, &arg_exit_idle_time);
+                        if (r < 0) {
+                                log_error("Failed to parse --exit-idle-time= argument: %s", optarg);
+                                return r;
+                        }
```

Comment:
```C
if (r < 0)
     return log_error_errno(r, "Failed to parse --exit-idle-time= argument: %s", optarg);
```

Reply:
Interesting. I had been mirroring `--connections-max=`, but that is convenient (as long as people accept it as "normal").